ufmt/core.py

```diff
@@ -29,27 +36,57 @@ class Result:
     diff: Optional[str] = None
 
 
-def ufmt_string(path: Path, content: str, config: Config) -> str:
-    content = usort_string(content, config, path)
-    mode = Mode()  # TODO
+def ufmt_string(
+    path: Path,
+    content: str,
+    usort_config: UsortConfig,
+    black_config: Optional[Mode] = None,
```

Comment:
Depending if you want to keep BC or not, we can also remove the default.

Reply:
I'm fine with this as-is.